packages/core/src/routes/session/index.ts

```diff
@@ -19,6 +20,7 @@ import { getRoutePrefix } from './utils';
 export default function sessionRoutes<T extends AnonymousRouter>(router: T, provider: Provider) {
   router.use(getRoutePrefix('sign-in'), koaGuardSessionAction(provider, 'sign-in'));
   router.use(getRoutePrefix('register'), koaGuardSessionAction(provider, 'register'));
+  router.use(getRoutePrefix('forgot-password'), koaGuardSessionAction(provider));
```

Comment:
After consideration, we can remove this for now, and bring it back until we have a control switch in AC for "forgot-password" (just like the registration switch)

Reply:
updated